services/security/identity/identity.go

```diff
@@ -9,15 +9,29 @@ import (
 	"github.com/microsoft/moc/pkg/errors"
 	"github.com/microsoft/moc/pkg/status"
 	wssdcloudsecurity "github.com/microsoft/moc/rpc/cloudagent/security"
+	wssdcloudcommon "github.com/microsoft/moc/rpc/common"
 )
 
 func getIdentity(id *wssdcloudsecurity.Identity) *security.Identity {
+	clitype := security.ExternalClient
+	if id.ClientType == wssdcloudcommon.ClientType_CONTROLPLANE {
+		clitype = security.ControlPlane
+	} else if id.ClientType == wssdcloudcommon.ClientType_EXTERNALCLIENT {
```

Comment:
ExternalClient and Node is not same right?

Reply:
good catch. This should be checking for ClientType_NODE, and external client should be default (external client will have faster expiry/more restrictions in the future).